python_int/title_encoder.py

```diff
@@ -1,9 +1,49 @@
-with open("python_core/books.txt", "r") as file:
+with open("/mnt/c/Users/wahrh/Documents/code/sololearn_snek/python_core/books.txt", "r") as file:
     raw = file.readlines()
     raw_lst = [line.strip() for line in raw]
-    print(raw_lst)
+    new_lst = []
```

Comment:
Kludgey. Gonna see if I can [golf](https://codegolf.stackexchange.com/questions/15279/tips-for-golfing-in-bash) this in main